apps/backend/src/modules/users/user.service.ts

```diff
@@ -55,6 +66,38 @@ class UserService implements Service {
 	public update(): ReturnType<Service["update"]> {
 		return Promise.resolve(null);
 	}
+
+	public async updateAvatar(
+		userId: number,
+		newAvatarFile: MultipartFile,
+	): Promise<null | UserEntity> {
+		const user = await this.userRepository.find(userId);
+
+		if (!user) {
+			throw new Error("User not found");
```

Comment:
We should throw a special error fot this module:
```
throw new UserError({
				message: ErrorMessage.<message>,
				status: HTTPCode.<code>,
			});
```
(Check createUserAnswers in quiz-answers.service.ts as an example)